docs/development/pigs/pig-013.rst

```diff
@@ -0,0 +1,271 @@
+matplotlib       2.1   (Oct 2017)  3.0   (Sep 2018)
+pandas           0.24  (Jan 2019)  0.25  (Jul 2019)
+healpy           1.11  (Aug 2017)  1.12  (Jun 2018)
+reproject        0.4   (Jan 2018)  0.5   (Jun 2019)
+uncertainties    3.0   (Aug 2016)  3.1   (May 2019)
+iminuit          1.3   (Jul 2018)  1.3.7 (Jun 2019)
+sherpa           4.11  (Feb 2019)  4.11  (Feb 2019)
+naima            0.8   (Dec 2016)  0.8.3 (Nov 2018)
+emcee            --                2.2   (Jul 2016)
+corner           --                2.0   (May 2016)
+parfive          --                1.0   (May 2019)
+===============  ================  ================
+
+We plan to use ``parfive`` for tutorial notebook and example dataset file
+download (features parallel download and a progress bar).
+
+Distributions
+=============
+
+The situation concerning distribution was described in the introduction above,
+and some alternatives and future work are mentioned in the outlook and
+alternatives sections below.
+
+We propose to remove the Macports installation page from the Gammmapy
+documentation, and to just leave a mention that it's outdated and unsupported on
+the page on "other ways to install Gammapy". Short sections or pages about pip
+and Debian will remain at the end of the installation instructions, for advanced
+users.
+
+For conda, we will improve the installation instructions, explaining the
+difference between using the recommended environment and a plain ``conda install
+-c conda-forge gammapy``, and add lists of required and optional dependencies as
+above to the Gammapy installation documentation.
+
+Outlook
+=======
+
+This PIG describes only the status and very short term plan for Gammapy v0.14
+and v1.0! We expect that dependencies and distribution will evolve in the coming
+years.
+
+E.g. we might add Numba as a dependency, or we might start to fully support pip,
+if binary wheels for Gammapy and all dependencies are available (there is a
+recent effort to implement this).
+
+Somewhat related to this PIG: we plan to modernise Gammapy packaging following
+the recommendations in `APE 17`_. Work on this has already started in `GH
+2279`_.
+
+Alternatives
+============
+
+Since we have a working setup already, we could do nothing, which seems nice at
+first, but really means continued time sink and suffering by the Gammapy
+maintainers of the continuous integration testing system and the release
+manager.
+
+For conda, we should strongly consider introducing a ``gammapy-all`` metapackage
+instead of the current ``gammapy-environment.yml`` shipped via ``gammapy.org``.
+That's what e.g. the Fermi tools and glueviz and others do, and they like that
+solution. It does have the advantage ob using a conda-native solution to the
+question how to ship an environment. This could be prototyped any time, in
+parallel with the existing solution, to gain familiarity with metapackages.
+
+We could support other ways to ship Gammapy, e.g. Homebrew (works on MacOS and
```

Comment:
I think this rather belongs to the outlooks section. It's not really an alternative, but rather optional.

Reply:
done